### ISSUE TITLE
Text in splash screen shows up late

### ISSUE THREAD

**Issue by grenzionky** [godotengine/godot#25466]
**Godot version:**
3.0.6 (i tried 32 and 64 bit of both mono and non mono version and the bug exists in all of them)

**OS/device including version:**
Windows 10

**Issue description:**
when tying text into any text area in the splash screen (including popups like the built in file manager) the text only shows up after tying a second character. So for example if i were to type "godot" i would see the 'g' only after tying the 'o' and the 'o' only after typing the 'd' and so on.

**Comment by groud** [comment]
Have you tried with 3.1 ?

**Comment by grenzionky** [comment]
just did. still there

**Comment by groud** [comment]
I cannot reproduce on Linux

**Comment by akien-mga** [comment]
Likely duplicate of #23069.

**Comment by grenzionky** [comment]
> Likely duplicate of #23069.

that also happens to me

**Comment by akien-mga** [comment]
I've edited the title to remove the mention of a specific system, but the title is the accurate description of the bug: everything is drawn one frame behind.

Closing as duplicate of #23069.

**Comment by akien-mga** [comment]
For the reference, an upcoming Intel drivers update *should* fix it, probably in February.
